File: sandbox-runner/patterns/local-activities/java/src/main/java/Worker.java
import io.temporal.client.WorkflowClient;
import io.temporal.serviceclient.WorkflowServiceStubs;
import io.temporal.worker.WorkerFactory;

public class Worker {
    public static void main(String[] args) {
        WorkflowServiceStubs service = WorkflowServiceStubs.newLocalServiceStubs();
        WorkflowClient client = WorkflowClient.newInstance(service);
        WorkerFactory factory = WorkerFactory.newInstance(client);

        io.temporal.worker.Worker worker = factory.newWorker(Shared.TASK_QUEUE);
        worker.registerWorkflowImplementationTypes(TransactionWorkflow.Impl.class);
        worker.registerActivitiesImplementations(new Activities.Impl());

        System.out.println("Worker started on task queue: " + Shared.TASK_QUEUE);
        factory.start();
    }
}

File: sandbox-runner/patterns/early-return-local-activities/python/worker.py
import asyncio
from temporalio.client import Client
from temporalio.worker import Worker
from activities import validate_transaction, reserve_funds, settle_transaction, cancel_transaction
from workflows import TransactionWorkflow
from shared import TASK_QUEUE


async def main() -> None:
    client = await Client.connect("localhost:7233")
    worker = Worker(
        client,
        task_queue=TASK_QUEUE,
        workflows=[TransactionWorkflow],
        activities=[validate_transaction, reserve_funds, settle_transaction, cancel_transaction],
    )
    print(f"Worker started on task queue: {TASK_QUEUE}")
    await worker.run()


if __name__ == "__main__":
    asyncio.run(main())
